.gitlab/functional_test/oracle.yml

```diff
@@ -0,0 +1,20 @@
+oracle:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["runner:docker"]
+  stage: functional_test
+  needs: ["go_deps"]
+  services:
+    - alias: "oracle"
+      name: "registry.ddbuild.io/images/mirror/oracle:${DBMS_VERSION}"
+      variables:
+        ORACLE_PWD: "datad0g"
```

Comment:
Double checking for safety: this is clear on purpose, right?

Reply:
Yes, it's just a local testing container and cleartexting it makes the ops a lot easier.